projects/lights_control/inc/lights_config.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "gpio.h"
+
+typedef enum {
+  LIGHTS_CONFIG_BOARD_TYPE_FRONT = 0,
+  LIGHTS_CONFIG_BOARD_TYPE_REAR,
+  NUM_LIGHTS_CONFIG_BOARDS
+} LightsConfigBoardType;
+
+typedef struct LightsConfig {
+  const LightsConfigBoardType board_type;
+  const GPIOAddress *addresses;
+  const uint8_t num_addresses;
+  const GPIOSettings *gpio_settings_out;
+  const uint16_t (*event_mappings)[2];
+  const uint8_t num_supported_events;
```

Comment:
It helps to use the same naming scheme across your array/len pairs. For example, it's obvious that `addresses` and `num_addresses` and connected.